### ISSUE TITLE
508-defect-1 ❗ Launchblocker [FOCUS MANAGEMENT, SCREEN READER]: Focus SHOULD be managed

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#8788]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Focus **should** move from the Search button to the H1 on the search results page. If a search is updated on the results screen, then focus **should** move to the results heading. 

![Screen Shot 2020-04-06 at 10 52 04 AM](https://user-images.githubusercontent.com/57469/78590191-73bf9780-780f-11ea-9946-3cc2cad0168f.png)

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to know a result has been provided when I submit a form.

**Comment by kelsonic** [comment]
This is already implemented, closing:

![image](https://user-images.githubusercontent.com/12773166/81307544-05543c00-903e-11ea-8214-f0e68ebdbfe5.png)
